include/NovelRT/Timing/StepTimer.h

```diff
@@ -59,11 +57,11 @@ namespace NovelRT::Timing {
       _targetElapsedTicks = value;
     }
 
-    inline double getTargetElapsedSeconds() const {
+    inline Timestamp getTargetElapsedTime() const {
       auto targetElapsedTicks = getTargetElapsedTicks();
```

Comment:
Maybe should be part of the API review, but it might be better if `getTargetElapsedTicks` wasn't exposed to the user
and instead users just get the timestamp and call `getTicks` on that
The compiler will optimize it the same, but it encourages users to work with `Timestamp` instead, which is likely the "more correct" thing

Reply:
(same with the other `get*Ticks` methods in here)

Reply:
Marked it for discussion for the review.